resurfemg/postprocessing/features.py

```diff
@@ -8,7 +8,7 @@
 import numpy as np
 import logging
 from ..preprocessing.envelope import running_smoother
-from ..helper_functions import delay_embedding
+from resurfemg.helper_functions_legacy import delay_embedding
```

Comment:
delay_embedding is in the new helper_functions/helper_functions file --> resurfemg.helper_functions..helper_functions